packages/docz-tools/src/gatsby-theme-docz/components/Sidebar/styles.ts

```diff
@@ -1,6 +1,6 @@
 export const sidebar = (width: number, sidebarOffset: number) =>
   ({
-    bg: "greyBlueDark",
+    bg: "blue",
```

Comment:
These .ts files do not use the semantic color values - this replaces old surface-reverse equivalent with new surface-reverse equivalent